README.md

```diff
@@ -1,3 +1,38 @@
 # polyglotweekly.com
 
-Curated articles on the art of programming.
+Curated articles about the art of programming.
+
+## How Polyglot Weekly Works
+
+### Generating an Article
+
+Anyone can submit a pull request for an article in Polyglot Weekly, simply run:
+
+```shell
+git clone https://github.com/polyglotweekly/polyglotweekly.com.git
+cd polyglotweekly.com
+nvm use nvm use 0.10 # sorry, no 0.12.x or iojs yet.
```

Comment:
Not a huge `nvm` expert, but this call doesn't seem right, I think it's just `nvm use 0.10`.